neo4j/internal/bolt/message_queue_test.go

```diff
@@ -76,6 +76,7 @@ func TestMessageQueue(outer *testing.T) {
 				description: "ROUTE",
 				action: func(queue *messageQueue) {
 					queue.appendRoute(nil, nil, nil, handlers[msgRoute])
+					// note: appendRouteV43 is also ROUTE, so skipping it
```

Comment:
Why does it exists then if it's exactly the same?

Reply:
it's not the same, but it uses the same message marker.
this test suite only asserts message markers and does not check content to keep things simple (else I'll end up coupling the test assertions with the packstream serialization logic)